src/encord_active/lib/model_predictions/map_mar.py

```diff
@@ -170,7 +170,7 @@ def compute_mAP_and_mAR(
         full_list_to_class_level_idx: Dict[int, int] = {v.item(): i for i, v in enumerate(class_level_to_full_list_idx)}
 
         _ious = ious[pred_select]
-        TP_candidates = set(class_level_to_full_list_idx[_ious >= iou_threshold].astype(int).tolist())
+        TP_candidates = set(class_level_to_full_list_idx[_ious >= (iou_threshold or 0)].astype(int).tolist())
```

Comment:
Not sure why 0 is the choice here? 0 means that everything will be considered, even predictions that don't overlap with any labels. I would at least add a small epsilon I think.

Reply:
I reverted it back to using the default we had before this. Now the UI will always show the IOU slider.

Reply:
Not sure I 100% understand the implications then?

Reply:
Basically, the UI will show the IOU slider with the original default and it will affect the color or the predictions. On top of that, you can filter to only see F/TP or FN.